swap_setup.sh

```diff
@@ -1,16 +1,20 @@
 #!/bin/bash
 
+IMAGE_SIZE=${1:-1} # set to default of 1GB if not passed in command line
+
+IMAGE_SIZE=${1:-1} # set to default of 1GB if not passed in command line
```

Comment:
Why are you setting this twice?

Reply:
That's a typo leftover form rebasing. I'll fix it